src/lib/methods/insert-item.ts

```diff
@@ -72,6 +75,17 @@ export class InsertItem extends Method implements Executable {
 		return result;
 	}
 
+	/**
+	 * Configure the retry policy
+	 *
+	 * @param opts - Retry configuration options.
+	 */
+	retry(opts: Options) {
```

Comment:
Is this something we can move to the `Method` class? As it feels like it's part of every database method?

Reply:
Done and the `runQuery` utility function as well.